### PR TITLE
Update screen-share.mdx

### PR DESCRIPTION
**Pull request by pawan-100ms**

### CODE REVIEW

File: docs/ios/v2/features/screen-share.mdx [hunk 1]
```diff
@@ -122,6 +124,15 @@ When a user leaves the meeting room from the main app, you will receive an error
         ...
     }
   ```
+  
+## Debugging issues in your screen-share implementation
+
+1. If you use cocoapods with M1 mac and compile for simulator then your might run into an error like this: "error framework not found Pods_hmsscreenshare"
+This is an issue if you use cocoapods for adding 100ms SDK and works fine if you are using swift package manager. As a workaround please use a snippet similar to at the end of this example podfile to exclude arm64 arch for simulator from your pod screen share target:
```
Comment (gzerad) [inline docs/ios/v2/features/screen-share.mdx:131]: Id simplify this a bit:
This is an issue if you use cocoapods for adding 100ms SDK and works fine if you are using swift package manager. => This issue is specific to cocoapods method of integrating the 100ms SDK.

File: docs/ios/v2/features/screen-share.mdx [hunk 2]
```diff
@@ -122,6 +124,15 @@ When a user leaves the meeting room from the main app, you will receive an error
...
+Workaround for M1 simulator issue https://github.com/100mslive/100ms-ios-sdk/blob/main/Example/Podfile
+
+2. If you are getting a crash while running with screen share feature, please make sure that the app group string is not misspelled in both the main app and the broadcast extension. Also make sure your entitlement file has the correct app group name.
```
Comment (gzerad) [inline docs/ios/v2/features/screen-share.mdx:134]: the app group string => app group identifier

File: docs/ios/v2/features/screen-share.mdx [hunk 3]
```diff
@@ -122,6 +124,15 @@ When a user leaves the meeting room from the main app, you will receive an error
         ...
     }
   ```
+  
+## Debugging issues in your screen-share implementation
+
+1. If you use cocoapods with M1 mac and compile for simulator then your might run into an error like this: "error framework not found Pods_hmsscreenshare"
```
Comment (gzerad) [inline docs/ios/v2/features/screen-share.mdx:130]: your => you

File: docs/ios/v2/features/screen-share.mdx [hunk 4]
```diff
@@ -5,6 +5,8 @@ nav: 3.981
 
 iOS SDK provides support for sharing the entire screen of the device in the room.
 
+It uses Replaykit framework from Apple. To understand the documentation better consider familiarising yourself with live screen broadcast in iOS: https://developer.apple.com/videos/play/wwdc2018/601
```
Comment (gzerad) [inline docs/ios/v2/features/screen-share.mdx:8]: maybe link replay kit documentation url as well?

File: docs/ios/v2/features/screen-share.mdx (hunk 1 above)
Comment (pawan-100ms) [reply docs/ios/v2/features/screen-share.mdx]: Rephrased but also left a mention of SPM just so people can switch of they want

**Review by gzerad — APPROVED** [review]

**Pull request merged**